examples/README.md

```diff
@@ -9,25 +9,111 @@ This collection of sample templates will help you get started with MongoDB Atlas
+To give the `execution role` access to the CMK, you must configure a [KMS Key Policy](https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html).
+The recommended options include:
+- [Allow access via Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/security-encryption.html#security-encryption-policies):
+  ```json
+  {
+    "Statement": [
+        // other statements
+                {
+            "Sid": "Enable IAM User Permissions",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::123456789:root"
+            },
+            "Action": "kms:*",
+            "Resource": "*"
+        },
+        {
+            "Action": [
+                "kms:Decrypt"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "kms:CallerAccount": "123456789",
+                    "kms:ViaService": "secretsmanager.{YOUR_REGION}.amazonaws.com"
+                }
+            },
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::123456789:role/YOUR_CFN_EXECUTION_ROLE" // or "*" to allow all of your IAM user/roles
+                ]
+            },
+            "Resource": "*",
+            "Sid": "Allow access through AWS Secrets Manager for the CFN Execution Role"
+        }
+    ],
+    "Version": "2012-10-17"
+  }
+  ```
+  - replace `123456789` with your [AWS account ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html)
+  - replace `{YOUR_REGION}` with the CFN region, e.g., `us-east-1`
+- Use `kms:decrypt` on the `execution role` and [enable IAM policies](https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-default.html#key-policy-default-allow-root-enable-iam) on the KMS key:
+  ```json
+  {
+    "Statement": [
+    // other statements
+        {
+            "Sid": "Enable IAM User Permissions",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::123456789:root"
+            },
+            "Action": "kms:*",
+            "Resource": "*"
+        }
+    ],
+    "Version": "2012-10-17"
+    }
+  ```
+  - replace `123456789` with your [AWS account ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html)
+  - ensure the IAM Execution Role has `kms:decrypt` permission:
+  ```json
+  {
+    "Statement": [
+        // other statements, e.g., `secretsmanager:GetSecretValue`
+        {
+            "Action": "kms:Decrypt",
+            "Effect": "Allow",
+            "Resource": "{KEY_ARN}"
+        }
+    ],
+    "Version": "2012-10-17"
+  }
+  ```
+  - replace `{KEY_ARN}` with your kms key arn
 
 ## Using the examples
 Once your prerequisites are configured, use the examples in this folder as a starting template for a resource to quickly create a CloudFormation stack. 
```

Comment:
Can you double check if below cluster.json is correct? Believe since we are missing Database User connection string will fail to create. Also it needs an IP Access List entry otherwise it will be closed by default and no one with connection string will be able to access.

Reply:
It was referring to the wrong template file.
Fixed in bd031a1f12c597a35fc21304d745b90b16347cc6